tasks/Task.hpp

```diff
@@ -46,6 +47,7 @@ namespace canbus {
 
         base::Time m_can_check_interval;
         base::Time m_stats_interval;
+        std::vector<canbus::CanOutputPort> outputports;
```

Comment:
You do not need to declare this as a class attribute anymore. Make it a local variable in the configureHook